README.md

```diff
@@ -1,96 +1,56 @@
-To ensure proper support of this feature, each release of ANTLR is a complete release of the tool and the 10 runtimes, all with the same version.
-As such, ANTLR versioning does not strictly follow semver semantics:
-
-* a component may be released with the latest version number even though nothing has changed within that component since the previous release
-* major version is bumped only when ANTLR is rewritten for a totally new "generation", such as ANTLR3 -> ANTLR4 (LL(\*) -> ALL(\*) parsing)
-* minor version updates may include minor breaking changes, the policy is to regenerate parsers with every release (4.11 -> 4.12)
-* backwards compatibility is only guaranteed for patch version bumps (4.11.1 -> 4.11.2)
-
-If you use a semver verifier in your CI, you probably want to apply special rules for ANTLR, such as treating minor change as a major change.
+WebAssembly is still being bleeding edge, and the 1st version of ANTLR5 will only support TypeScript.
+As other platforms provide support for recent WebAssembly features, such as garbage collection and exception handling, ANTLR5 will rapidly become available for these platforms.
 
 ## Repo branch structure
 
-The default branch for this repo is [`master`](https://github.com/antlr/antlr4/tree/master), which is the latest stable release and has tags for the various releases; e.g., see release tag [4.9.3](https://github.com/antlr/antlr4/tree/4.9.3).  Branch [`dev`](https://github.com/antlr/antlr4/tree/dev) is where development occurs between releases and all pull requests should be derived from that branch. The `dev` branch is merged back into `master` to cut a release and the release state is tagged (e.g., with `4.10-rc1` or `4.10`.) Visually our process looks roughly like this:
+The default branch for this repo is [`main`](https://github.com/antlr/antlr5/tree/main), which is the latest stable release and has tags for the various releases.  Branch [`dev`](https://github.com/antlr/antlr5/tree/dev) is where development occurs between releases and all pull requests should be derived from that branch. The `dev` branch is merged back into `main` to cut a release and the release state is tagged (e.g., with `1.0-rc1` or `1.0`.) Visually our process looks roughly like this:
 
 <img src="doc/images/new-antlr-branches.png" width="500">
 
-The Go target now has its own dedicated repo:
-
-```bash
-$ go get github.com/antlr4-go/antlr
-```
-**Note**
-The dedicated Go repo is for `go get` and `import` only. Go runtime development is still performed in the main `antlr/antlr4` repo. 
-
 ## Authors and major contributors
+We're only providing here the list of ANTLR 5 contributors. ANTLR 5 is largely based on ANTLR 4. See [ANTLR 4](https://github.com/antlr/antlr4) for the list of ANTLR 4 contributors, and they are recognized as silent ANTLR 5 authors.
 
 * [Terence Parr](http://www.cs.usfca.edu/~parrt/), parrt@cs.usfca.edu
 ANTLR project lead and supreme dictator for life
```

Comment:
I suggest removing because he is not the lead of ANTLR 5 and he doesn't participate. We have references to ANTLR 4, it's enough. Also, "dictator for life" looks quite strange.

Reply:
See https://en.wikipedia.org/wiki/Benevolent_dictator_for_life (except Ter is not benevolent).
Since only the dictator can opine on this, I'll leave it to him...

Reply:
Maybe it's worth to make this title clickable with pointed link (people from other countries and cultures may not aware of it and may treat it in a wrong way).

Reply:
Good idea!